users/admin.py

```diff
@@ -8,9 +11,15 @@
 
 
 @admin.register(User)
-class UserAdminInline(VersionAdmin):
+class UserAdminInline(VersionAdmin, DjangoUserAdmin):
     inlines = [ProfileAdminInline]
 
+    def get_fieldsets(self, request, obj=None):
+        fieldsets = deepcopy(super().get_fieldsets(request, obj))
+        fieldsets[1][1]["fields"] += ("uuid",)
```

Comment:
I get this while trying to create a new user:

```
Exception Type: | IndexError
-- | --
tuple index out of range
/app/users/admin.py in get_fieldsets, line 19
```

Reply:
Yeah, apparently forgot to test user creation with the newest iteration of the thing. Should be fixed now.